vcpkg/users/github-dependency-graph.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Integrating with GitHub dependency graph
+description: Record the set of resolved libraries with the GitHub dependency graph
+ms.date: 05/22/2023
+---
+
+# Integrating with the GitHub dependency graph
+
```

Comment:
Needs to add the experimental block, see how it is done for the asset caching docs